app/src/main/java/org/oppia/app/profile/ProfileChooserViewModel.kt

```diff
@@ -0,0 +1,10 @@
+package org.oppia.app.profile
+
+import androidx.lifecycle.ViewModel
+import org.oppia.app.fragment.FragmentScope
+import javax.inject.Inject
+
+/** The ViewModel for [ProfileChooserFragment] */
+@FragmentScope
+class ProfileChooserViewModel @Inject constructor(): ViewModel() {
```

Comment:
This can actually be an `ObservableViewModel` instead of `ViewModel`, we are now using `ObservableViewModel` instead of `ViewModel` here and elsewhere.